dvc/command/stage.py

```diff
@@ -348,7 +348,7 @@ def add_parser(subparsers, parent_parser):
         "--fail",
         action="store_true",
         default=False,
-        help="Fail immediately if there's an error.",
+        help="Do not suppress any syntax errors, fail immediately.",
```

Comment:
Again nitpicking, but I'd suggest switching the order so the primary effect of the flag is obvious from the first couple words.

```suggestion
        help="Fail immediately, do not suppress any syntax errors.",
```